agent/installer/installer_test.go

```diff
@@ -34,10 +34,10 @@ var _ = Describe("Byohost Installer Tests", func() {
 			for _, os := range ListSupportedOS() {
 				i := NewPreviewInstaller(os, nil)
 
-				err := i.Install("unsupported-k8s")
+				err := i.Install("unsupported-k8s", "0.1.0")
```

Comment:
nit: In the tests, can we have tag values like `test-tag`, `dummy-tag` etc.? Because looking at `0.1.0`, it feels we are enforcing tag to be of this format.
I prefer the way we are having `unsupported-k8s` instead of values like `v1.22.0`